lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -87,6 +89,15 @@ def main(argv=None):
                              'ensemble mean ("mean") and the ensemble '
                              'realizations ("realizations") are supported as '
                              'options. Default: "mean".')
+    parser.add_argument('--max_iterations', metavar='MAX_ITERATIONS',
+                        type=np.int32, default=200,
+                        help='The maximum number of iterations allowed '
```

Comment:
It might be worth saying in this docstring that the number of iterations should be increased if you are using realizations as a predictor.

Reply:
Still outstanding @gavinevans

Reply:
I've now extended this docstring, @fionaRust.

Reply:
This should probably be copied into the other CLI too. Sorry